src/main/webapp/ProfilePage.html

```diff
@@ -1,38 +1,41 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-    />
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="user_script.js"></script>
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-    />
-  </head>
-  <body onload="loadProfile()">
-    <div id="navbar"></div>
-    
-    <div class="container">
-      <h1>My Profile Page</h1>
-      <div class="card" style="width: 22rem;">
-        <img id ="avatar" src="" class="card-img-top" alt="profile avatar">
-        <div class="card-body">
-          <h5 class="card-title" id="name"> </h5>
-          <h6 class="card-subtitle mb-2 " id="username"> </h6>
-          <p class="card-text" id="bio"> </p>
-        </div>
+
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
```

Comment:
Make sure to format the file for proper indentation using prettier

Reply:
I did use prettier. I think it looks different on github

Reply:
can you try prettier again and push the change? Theres still some formatting issues.

Reply:
I have reformatted the code in Profile Page but it keeps looking wonky in github.

Reply:
Hmm can you try `npx prettier --write ~/step110-2020/src/main/webapp/ProfilePage.html --tab-width 2`

This should hopefully force the tab size to be 2 spaces

Reply:
That worked! Thank you !

Reply:
Yay! You can also run that for EditProfilePage.html since the formatting there is a little off too.

`npx prettier --write ~/step110-2020/src/main/webapp/EditProfilePage.html --tab-width 2`